frontend/src/components/UserSettings/UserInformation.tsx

```diff
@@ -58,12 +58,14 @@ const UserInformation = () => {
     },
     onError: (err: ApiError) => {
       const errDetail = (err.body as any)?.detail
-      showToast("Something went wrong.", `${errDetail}`, "error")
+      let errorMessage = "Something went wrong."
+      if (Array.isArray(errDetail) && errDetail.length > 0) {
+        errorMessage = errDetail[0].msg
+      }
+      showToast("Error", errorMessage, "error")
     },
     onSettled: () => {
-      // TODO: can we do just one call now?
-      queryClient.invalidateQueries({ queryKey: ["users"] })
-      queryClient.invalidateQueries({ queryKey: ["currentUser"] })
+      queryClient.invalidateQueries()
```

Comment:
this invalidates all the queries, right?

Reply:
Yes :nerd_face: